analysis_options.yaml

```diff
@@ -8,7 +8,7 @@ analyzer:
     # When compiling to JS, both implicit options apply to the current 
     # project and all dependencies. They are useful to find possible 
     # Type fixes or areas for explicit typing.
-    implicit-casts: true
+    implicit-casts: false
```

Comment:
I think we should use the replacements available since 2.16 https://dart.dev/tools/analysis#enabling-additional-type-checks

```
analyzer:
  language:
    strict-casts: true
```

Reply:
Will this replace:
```
analyzer:
    strong-mode:
        implicit-casts: true
        implicit-dynamic: true
```

or just `implicit-casts: true`

Reply:
It would replace the whole strong-mode block. 
implicit-dynamic: true is the default and doesn't do anything, so can safely be removed.